### PR TITLE
Provider authentication identity

### PR DESCRIPTION
**Pull request by mrclemds**
Add Authentication

Update Identity branch
Add a service to manage Auth
Change popin link

Ref to [this ticket](https://trello.com/c/16lrvWNF/7-histoire-cr%C3%A9er-un-compte-prestataire)

### CODE REVIEW

File: src/app/modules/timesheet/components/timesheet-edit/timesheet-edit.component.html [hunk 1]
```diff
@@ -1,160 +1,164 @@
...
+    </div>
+  </div>
+
+  <div class="edit-cra__wrapper">
+    <div class="edit-cra__wrapper__title">
+      <h2>{{ getModeTitle() }} un compte rendu d'activité</h2>
+    </div>
+
+    <form class="edit-cra__wrapper__form" #form="ngForm">
+      <div id="prestataire" class="edit-cra__wrapper__form__block">
```
[the remark this answers is not in the document]
Comment (pfilippi) [reply src/app/modules/timesheet/components/timesheet-edit/timesheet-edit.component.html]: and choose the right one ;-)

File: package.json [hunk 2]
```diff
@@ -33,6 +33,7 @@
     "date-fns": "^1.30.1",
     "material-icons": "^0.2.3",
     "moment": "^2.24.0",
+    "netlify-identity-widget": "github:iteatime/netlify-identity-widget#staging",
```
Comment (RomainOliva) [inline package.json]: Staging is just for testing, you will have some normal tag after ?

**Comment by RomainOliva** [comment]
I think you have problem on this branch, this PR is for merge your branch into `cpont-refactor-timesheet-components`, but you have every `cpont-refactor-timesheet-components` commits, you did a rebase on staging or something like that no ?

File: package.json (hunk 2 above)
Comment (mrclemds) [reply package.json]: Yes

**Comment by pfilippi** [comment]
It would be nice to be able to Log In / Out anywhere in the app, mostly on the timesheet review page

**Pull request merged**